### PR TITLE
Updated the Tile form field descriptions to be more informative

### PR DESCRIPTION
**Pull request by keerthibokka**

### CODE REVIEW

File: pcf-tile-metadata/aci-cni-plugin.yml.template [hunk 1]
```diff
@@ -137,108 +137,108 @@ property_blueprints:
...
   - reference: ".properties.apic_vmm_controller"
-    label: Apic Vmm Controller
-    description: data_type:string
+    label: APIC VMM Controller
+    description: APIC VMM controller to use
   - reference: ".properties.apic_name_prefix"
-    label: Apic name prefix
-    description: data_type:string
+    label: APIC Name Prefix
+    description: String to prefix to APIC objct names
```
Comment (amittbose) [inline pcf-tile-metadata/aci-cni-plugin.yml.template]: missing 'e' in "object"

File: pcf-tile-metadata/aci-cni-plugin.yml.template [hunk 2]
```diff
@@ -137,108 +137,108 @@ property_blueprints:
...
   - reference: ".properties.apic_private_key"
-    label: Apic private key
-    description: data_type:text
+    label: APIC Private Key
+    description: PEM encoded private key of user for connecting to APIC
   - reference: ".properties.apic_app_profile"
-    label: string
-    description: data_type:string
+    label: APIC App Profile
+    description: Application profile in APIC to use for isolation segments etc
```
Comment (amittbose) [inline pcf-tile-metadata/aci-cni-plugin.yml.template]: This should be "Application profile in APIC to use for endpoint groups"

File: pcf-tile-metadata/aci-cni-plugin.yml.template [hunk 3]
```diff
@@ -137,108 +137,108 @@ property_blueprints:
...
   - reference: ".properties.apic_service_phys_domain"
-    label: Apic Service Physdomain
-    description: data_type:string
+    label: APIC Service Physcal Domain
+    description: Physical domain used for service device clusters
   - reference: ".properties.apic_encap_type"
-    label: Apic Vmm encap type
-    description: data_type:string
+    label: APIC Encap type
+    description: Encap used in APIC for app external-ip device clusters
```
Comment (amittbose) [inline pcf-tile-metadata/aci-cni-plugin.yml.template]: This should be: "Encapsulation type (Vlan/VxLan) to use for container traffic"

File: pcf-tile-metadata/aci-cni-plugin.yml.template [hunk 4]
```diff
@@ -137,108 +137,108 @@ property_blueprints:
...
   - reference: ".properties.apic_service_encap"
-    label: Apic service encap
-    description: data_type:string
+    label: APIC Service Encap
+    description: Encap used in APIC for app external-ip device clusters
```
Comment (amittbose) [inline pcf-tile-metadata/aci-cni-plugin.yml.template]: This should be: "Vlan ID to use for service traffic"

File: pcf-tile-metadata/aci-cni-plugin.yml.template [hunk 5]
```diff
@@ -137,108 +137,108 @@ property_blueprints:
...
-    description: data_type:network_address
+    label: App External Static IP pool starting address
+    description: IP address pool to use for statically assigned app external IPs
   - reference: ".properties.network_app_static_ext_ip_pool_end"
-    label: IP pool end address
-    description: data_type:network_address
+    label: App External Static IP pool ending address
+    description: IP address pool to use for statically assigned app external IPs
   - reference: ".properties.network_node_service_ip_pool_start"
     label: IP pool start address
```
Comment (amittbose) [inline pcf-tile-metadata/aci-cni-plugin.yml.template]: This should be "Service IP Pool start address"

File: pcf-tile-metadata/aci-cni-plugin.yml.template [hunk 6]
```diff
@@ -137,108 +137,108 @@ property_blueprints:
...
-    description: data_type:network_address
+    description: IP address pool to use for cells to support app external IP
   - reference: ".properties.network_node_service_ip_pool_end"
     label: IP pool end address
```
Comment (amittbose) [inline pcf-tile-metadata/aci-cni-plugin.yml.template]: "Service IP Pool end address"

**Comment by keerthibokka** [comment]
Made the suggested changes

**Pull request merged**